src/OpenTelemetry.Contrib.Instrumentation.EntityFrameworkCore/OpenTelemetry.Contrib.Instrumentation.EntityFrameworkCore.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
```

Comment:
We should add `<IncludeSharedInstrumentationSource>true</IncludeSharedInstrumentationSource>`. This project was depending on the SDK before to create `TracerProviderSdk`.